clients/v1_auth/v1_auth.go

```diff
@@ -99,7 +99,8 @@ func (c Client) Create(ctx context.Context, params *CreateParams) error {
 
 	var permissions []api.Permission
 	for _, bp := range bucketPerms {
-		for _, p := range bp.perms {
+		for _, _p := range bp.perms {
+			p := _p
```

Comment:
This would be more idiomatic IMO, and would prevent accidental references to the wrong `p` value:
```suggestion
		for _, p := range bp.perms {
			p := p
```

Reply:
thanks for your review!!
I updated the commit for your suggestion.